### PR TITLE
Shadow DistSQL doc

### PR DESCRIPTION
**Pull request by lanchengx**
Changes proposed in this pull request:
- Shadow DistSQL doc

### CODE REVIEW

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+algorithmProperties: algorithmProperty [, algorithmProperty] ... 
+
+algorithmProperty: key=value
+```
+
+- 影子库压测请参考 [影子库压测](https://shardingsphere.apache.org/document/current/cn/features/shadow/)
+- `ruleName` 重复的名称无法被创建
+- `resourceMapping` 指定源数据库和影子库的映射关系，需使用 RDL 管理的`Resource`资源，请参考 [数据源资源](https://shardingsphere.apache.org/document/current/cn/features/dist-sql/syntax/rdl/rdl-resource/)
+- `shadowAlgorithm` 可同时作用于多个 `shadowTableRule`
+- `algorithmName`  未填写时会根据 `ruleName`、`tableName` 和 `shadowAlgorithmType` 自动生成（推荐数据库命名规范）
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md:36]: SQL statement is not a form, it is not suitable to use `填写`, `指定` may be better.

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+- `TYPE` 目前支持 `COLUMN_REGEX_MATCH` 和 `SIMPLE_NOTE`
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md:37]: `shadowAlgorithmType` 目前支持 `COLUMN_REGEX_MATCH` 和 `SIMPLE_NOTE`

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+- `shadowTableRule` 能够被不同的 `shadowRuleDefinition` 复用，因此在执行 `DROP SHADOW RULE` 时，对应的 `shadowTableRule` 不会被移除
+- `ALTER SHADOW RULE` 以 `ruleName` 作为修改条件，修改 `shadowTableRule` 时，该 `shadowRuleDefinition` 的已经存在的 `shadowTableRule` 会被覆盖
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md:39]: ‘该 `shadowRuleDefinition` 的已经存在的 `shadowTableRule` 会被覆盖’  
It is ambiguous.

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+shadowTableRule: tableName(shadowAlgorithm [, shadowAlgorithm] ...)
+
+shadowAlgorithm: ([algorithmName, ] TYPE(NAME=shadowAlgorithmType, PROPERTIES([algorithmProperties] ...)))
+
+algorithmProperties: algorithmProperty [, algorithmProperty] ... 
+
+algorithmProperty: key=value
+```
+
+- 影子库压测请参考 [影子库压测](https://shardingsphere.apache.org/document/current/cn/features/shadow/)
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.cn.md:32]: This sentence is a bit awkward

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+shadowTableRule: tableName(shadowAlgorithm [, shadowAlgorithm] ...)
+
+shadowAlgorithm: ([algorithmName, ] TYPE(NAME=shadowAlgorithmType, PROPERTIES([algorithmProperties] ...)))
+
+algorithmProperties: algorithmProperty [, algorithmProperty] ... 
+
+algorithmProperty: key=value
+```
+
+- `Shadow` please refer to [Shadow](https://shardingsphere.apache.org/document/current/cn/features/shadow/)
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md:32]: This sentence is a bit awkward

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md [hunk 6]
```diff
@@ -0,0 +1,65 @@
...
+- `ruleName` Duplicate name cannot be created
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md:33]: The order is not good, reference https://shardingsphere.apache.org/document/current/en/features/dist-sql/syntax/rdl/rdl-sharding-rule/

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md [hunk 7]
```diff
@@ -0,0 +1,65 @@
...
+- `resourceMapping` specifies the mapping relationship between the source database and the shadow library. You need to use the `Resource` resource managed by RDL, please refer to [Resources](https:shardingsphere.apache.orgdocumentcurrentcnfeaturesdist-sqlsyntaxrdlrdl-resource)
+- `shadowAlgorithm` can act on multiple `shadowTableRule` at the same time
+- If `algorithmName` is not filled in, it will be automatically generated according to `ruleName`, `tableName` and `shadowAlgorithmType` (recommended database naming convention)
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md:36]: `filled in` -> `specified `

File: docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md [hunk 8]
```diff
@@ -0,0 +1,65 @@
...
+- `TYPE` currently supports `COLUMN_REGEX_MATCH` and `SIMPLE_NOTE`
```
Comment (RaigorJiang) [inline docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md:37]: shadowAlgorithmType

Comment (lanchengx) [reply docs/document/content/features/dist-sql/syntax/rdl/rdl-shadow-rule.en.md]: All have been fixed

**Review by RaigorJiang — APPROVED** [review]

**Pull request merged**